### DISCUSSION TITLE
LineLayer not rendering on shortest path with repeat mode on across 180th Meridian

### DISCUSSION THREAD

**Discussion by Anand2489** [visgl/deck.gl#6954] [Q&A] [answered]
### Description

LineLayer is taking longer route to render with repeat mode on across 180th Meridian. 
Reference issue: https://github.com/visgl/deck.gl/issues/6950

### Expected Behaviour

Ideally LineLayer should render via the shorter route. Setting `wrapLongitude` to `true` splits the line into 2 as per the DeckGL [documentation](https://deck.gl/docs/api-reference/core/layer#wraplongitude) across 180th meridian

Any configuration to enable it ?

### Steps to Reproduce

<img width="1469" alt="Screenshot 2022-05-25 at 2 16 34 PM" src="https://user-images.githubusercontent.com/5299907/170223552-f7d63b1d-d14a-47f8-9a7b-90bac786fb5c.png">

### Code pen

https://codepen.io/andycool/pen/ExQbbeG?editors=001

<img width="1469" alt="Screenshot 2022-05-25 at 2 16 34 PM" src="https://user-images.githubusercontent.com/5299907/170369194-3c1969f9-4233-4350-a557-212929a9eb80.png">

**Comment by Pessimistress** [comment, marked as answer]
https://deck.gl/docs/api-reference/core/map-view#repeat

Same as https://github.com/visgl/deck.gl/discussions/6458
